package.json

```diff
@@ -2,7 +2,7 @@
   "name": "quoting-service",
   "description": "Quoting Service hosted by a scheme",
   "license": "Apache-2.0",
-  "version": "12.0.7",
+  "version": "12.0.8",
```

Comment:
It's time to automate this guy...

Reply:
So, I added `release` and `github-releases` into the CircleCI config.

Hope it works *holds thumbs*

Reply:
Yeah should do